qemu_v8.mk

```diff
@@ -492,6 +504,10 @@ run-only:
 		$(QEMU_XEN) \
 		$(QEMU_EXTRA_ARGS)
 
+.PHONY: run-only-with-scmi
+run-only-with-scmi: QEMU_EXTRA_ARGS+= -dtb $(SCMI_DTB)
+run-only-with-scmi: run-only
+
```

Comment:
Since `run*-with-scmi` end up calling `run` and `run-only`, I'd rather not add these new targets. SCMI should be treated like XEN for instance (`make run XEN_BOOT=y`). For example: `make run WITH_SCMI=y`. Another advantage is that you should get `make check WITH_SCMI=y` for free.

Reply:
ok, I'm going to have a look